src/temperature/crud.py

```diff
@@ -0,0 +1,33 @@
+from sqlalchemy import insert
+from sqlalchemy.future import select
+from sqlalchemy.ext.asyncio import AsyncSession
+from typing import List, Any, Optional
+from src.temperature.models import Temperature as TemperatureModel
+from src.temperature.schemas import TemperatureCreate
+
+
+async def create_temperature(
+    db: AsyncSession,
+    temperature: TemperatureCreate
+) -> dict[str, Any]:
+    query = insert(TemperatureModel).values(
+        city_id=temperature.city_id,
+        date_time=temperature.date_time,
+        temperature=temperature.temperature
+    ).returning(TemperatureModel.id)
+    result = await db.execute(query)
+    await db.commit()
+    resp = {**temperature.model_dump(), "id": result.lastrowid}
```

Comment:
The use of `model_dump()` on `temperature` assumes it is a Pydantic model with this method. Ensure that `TemperatureCreate` is a Pydantic model and supports `model_dump()`, or replace it with the appropriate method to convert the model to a dictionary.